frontend/packages/erd-core/src/features/erd/components/ERDContent/ErdContent.tsx

```diff
@@ -62,7 +62,8 @@ export const ERDContentInner: FC<Props> = ({
   const {
     state: { loading },
   } = useErdContentContext()
-  const { activeTableName } = useUserEditing()
+  const userEditing = useUserEditingOrThrow()
+  const { activeTableName } = userEditing
```

Comment:
[ask]

Can this be written in one line like this?

```suggestion
  const { activeTableName } = useUserEditingOrThrow()
```